src/main/webapp/mapScript.js

```diff
@@ -7,13 +12,15 @@ function initMap() {
     const mapViewDefault = {lat: 37.3868, lng: -122.085}; 
 
     const map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
you might want to use the closure library when interacting with your dom. In this case, goog.dom.getElement("map") should work